docs/iot/iot-partners/index.md

```diff
@@ -0,0 +1,27 @@
+# Tizen IoT with Nubison Cloud
+
+The Nubison cloud can readily be interconnected with Tizen IoT devices, which has an intrinsic capability of seamless connectivity.
+Real-time data monitoring and controlling between Tizen service app and Nubison cloud is empowered through Nubison plugin.
```

Comment:
It would be better to change as follow:

The Nubison IoT cloud is a cloud-based IoT platform that provides smart solutions and intelligent services  by connecting various things easily, collecting and analyzing IoT data in real time.

Tizen IoT devices contain a Nusion plugin, which supports real-time data monitoring and controlling between Tizen service app and Nubison cloud.

The following diagram shows the interface between Tizen IoT device and Nubison cloud through Nubision plugin.

Reply:
@journey2w : Yes, it is better and makes more sense. I changed the sentence slightly, also there was a typo error in "Nubison"  spelling, I corrected that too. 
Hope you are fine with the changes. 

The Nubison IoT cloud is a cloud-based IoT platform that provides smart solutions and intelligent services by connecting various things easily, also it collects and analyzes IoT data in real time.

Tizen IoT devices use Nubison plugin, which enables real-time data monitoring and controlling between Tizen service app and Nubison cloud.

The following diagram shows the interface between Tizen IoT device and Nubison cloud through Nubison plugin:

Reply:
The second sentence is incorrect as tizen iot devices DOES NOT contain a Nubison plugin... I think we'd better stick with the original sentence, which is 'Real-time data monitoring and controlling between Tizen service app and Nubison cloud is empowered through Nubison plugin.'

Reply:
My suggestion is as follows:

The Nubison IoT cloud is a cloud-based IoT platform that provides smart solutions and intelligent services by connecting various things easily, as well as collecting and analyzing IoT data in real time.

Real-time data monitoring and controlling between Tizen service app and Nubison cloud is empowered through Nubison plugin.

The following diagram shows the interface between Tizen IoT device and Nubison cloud through Nubison plugin:

Reply:
@bson1012 , thats why i changed @journey2w 's text slightly in my rephrasing 

The Nubison IoT cloud is a cloud-based IoT platform that provides smart solutions and intelligent services by connecting various things easily, also it collects and analyzes IoT data in real time.

Tizen IoT devices use Nubison plugin, which enables real-time data monitoring and controlling between Tizen service app and the Nubison cloud.

The following diagram shows the interface between Tizen IoT device and Nubison cloud through Nubison plugin: